### PR TITLE
Remove old hexadecimal and octal syntax.

### PR DESCRIPTION
**Pull request by Lojjs**

### CODE REVIEW

**Comment by Lojjs** [comment]
[Remove deprecated syntax for octals and hexadecimals](https://trello.com/c/POReMKfH/70-remove-deprecated-syntax-for-octals-and-hexadecimals)

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 1]
```diff
@@ -335,6 +335,16 @@ Replaced by:
...
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS YIELD allocatedBytes
+----
+a|
+This column has been removed as it wasn't tracked and always returned 0.
+
```
Comment (AzuObs) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:347]: Did you mean to include this? Doesn't seem related to Octal or Hex

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 2]
```diff
@@ -364,14 +374,26 @@ prop IS NULL
 ----
 
 a|
-label:functionality[]
+label:syntax[]
 label:removed[]
 [source, cypher, role="noheader"]
 ----
-SHOW TRANSACTIONS YIELD allocatedBytes
+0...
```
Comment (AzuObs) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:381]: Same question here. Suspicious!

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc (hunk 1 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:347]: Actually I just moved it up two steps to be together with the other SHOW TRANSACTION entries, but looks like Github is not presenting it very well

Comment (AzuObs) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:347]: Oh yes my bad :)

**Review by AzuObs — APPROVED** [review]

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 3]
```diff
@@ -335,6 +335,16 @@ Replaced by:
...
 
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS YIELD allocatedBytes
+----
+a|
+This column has been removed as it wasn't tracked and always returned 0.
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]:
```suggestion
The `allocatedBytes` output has been removed, because it was never tracked and thus was always 0.
```

**Review by martin-neotech — CHANGES_REQUESTED** [review]

**Review by martin-neotech — APPROVED** [review]
